projects/GroupDETR/group_detr/group_conditional_detr.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Dict, Tuple
+
+import torch
+import torch.nn as nn
+from torch import Tensor
+
+from mmdet.models.detectors.conditional_detr import ConditionalDETR
+from mmdet.models.layers.positional_encoding import SinePositionalEncoding
+from mmdet.models.layers.transformer.detr_layers import DetrTransformerEncoder
+from mmdet.registry import MODELS
+from .group_conditional_detr_decoder import \
+    GroupConditionalDetrTransformerDecoder
+
+
+@MODELS.register_module()
+class GroupConditionalDETR(ConditionalDETR):
+    r"""Implementation of `Fast DETR Training with Group-Wise
+     One-to-Many Assignment.
+
+    <https://arxiv.org/abs/2207.13085>`_.
+
+    Code is modified from the `official github repo
+    <https://github.com/Atten4Vis/ConditionalDETR/tree/GroupDETR>`_.
+
+    Args:
+        num_query_groups (int): The number of decoder query groups.
+    """
```

Comment:
should add **kwargs and tell users to refer to which module for the remaining arguments